x-pack/test/alerting_api_integration/observability/custom_threshold_rule/rate_bytes_fired.ts

```diff
@@ -46,9 +47,11 @@ export default function ({ getService }: FtrProviderContext) {
         schedule: [
           {
             template: 'good',
-            start: 'now-15m',
-            end: 'now',
-            metrics: [{ name: 'system.network.in.bytes', method: 'exp', start: 10, end: 100 }],
+            start: 'now-10m',
+            end: 'now+5m',
+            metrics: [
+              { name: 'system.network.in.bytes', method: 'linear', start: 0, end: 54000000 },
```

Comment:
I've adjusted the data generation schema and related time range (start, end) to fix the following issue:

![image](https://github.com/elastic/kibana/assets/12370520/0d141bc6-1266-4df5-becf-adaf90c36c81)

Special thanks to @simianhacker for helping with the math aspect of it!